csharp/autobuilder/Semmle.Autobuild/BuildActions.cs

```diff
@@ -96,12 +97,27 @@ public interface IBuildActions
         /// <returns>The combined path.</returns>
         string PathCombine(params string[] parts);
 
+        /// <summary>
+        /// Gets the full path for <paramref name="path"/>, Path.GetFullPath().
+        /// </summary>
+        string GetFullPath(string path);
+
+        /// <summary>
+        /// Gets the directory of <paramref name="path"/>, Path.GetDirectoryName().
+        /// </summary>
+        string GetDirectoryName(string path);
```

Comment:
This probably doesn't need to be stubbed out as it doesn't touch the filesystem.